aqi/algos/caqi.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+
+from decimal import *
+
+from aqi.constants import (POLLUTANT_PM25, POLLUTANT_PM10,
+                           POLLUTANT_O3_1H, POLLUTANT_CO_8H, POLLUTANT_SO2_1H,
+                           POLLUTANT_NO2_1H)
+from aqi.algos.base import PiecewiseAQI
+
+
+class AQI(PiecewiseAQI):
+    """Implementation of the EPA AQI algorithm.
```

Comment:
Fix the name here

Reply:
Name or the comment?
Because class name(AQI) seems to be used in get_algo implementation?
https://github.com/hrbonz/python-aqi/blob/1de807365eb44ef6c4e3d80af500ff8f6c273d41/aqi/algos/__init__.py#L20